src/helpers/load-package-json.ts

```diff
@@ -0,0 +1,14 @@
+import fs from "fs";
+import path from "path";
+
+export function loadPackageJson(): { [key: string]: any } {
+  let pkgContent = "{}";
+  try {
+    pkgContent = fs.readFileSync(
```

Comment:
This shouldn't be a sync operation

Reply:
And instead of caching the JSON string content, I would cache the parsed data.

Reply:
It's currently used from a getter, where async can't be used from, so there would some more code changes needed, but yes, that should be done in my opinion

Reply:
hmm. caching will need changing of default.test.ts

Reply:
If breaking changes are required, now is the time 😁

Reply:
Yeah, i think it is anyway internal api